package.json

```diff
@@ -55,5 +55,9 @@
     "sinon": "^2.3.7",
     "uuid": "^3.1.0",
     "windbreaker-service-util": "github:windbreaker-io/windbreaker-service-util"
+  },
+
+  "ava": {
+    "require": [ "require-self-ref" ]
```

Comment:
No need to add `require('require-self-ref')` to the top of every test file with this.